static/src/javascripts/projects/common/modules/experiments/tests/remove-business-liveblog-epics.ts

```diff
@@ -8,11 +8,11 @@ export const removeBusinessLiveblogEpics: ABTest = {
 	author: '@commercial-dev',
 	description:
 		'Test the commercial impact of removing contribution epics on business liveblogs',
-	audience: 0 / 100,
-	audienceOffset: 0 / 100,
-	audienceCriteria: 'Opt in',
+	audience: 20 / 100,
+	audienceOffset: 20 / 100,
+	audienceCriteria: 'Business liveblogs',
 	successMeasure: 'Ad revenue increases on business liveblogs',
-	canRun: () => true,
+	canRun: () => window.guardian.config.page.contentType === 'LiveBlog',
```

Comment:
If this test is only for business liveblogs, can we narrow this check down to business pages only? For example, either `window.guardian.config.page.section === 'business'` or `window.guardian.config.page.sectionName === 'Business'` maybe

Reply:
Good suggestion, have made the change!